api/methods/defaults.go

```diff
@@ -24,10 +24,11 @@ func GetDefaults(c *gin.Context) {
 		Code:    200,
 		Message: "success",
 		Data: gin.H{
-			"fqdn":            configuration.Config.FQDN,
-			"prometheus_path": configuration.Config.PrometheusPath,
-			"webssh_path":     configuration.Config.WebSSHPath,
-			"grafana_path":    configuration.Config.GrafanaPath,
+			"fqdn":               configuration.Config.FQDN,
```

Comment:
Here only `"valid_subscription"` line has been added, I don't understand why the 4 lines above it appear in the diff, even whitespace chars look the same :man_shrugging:

Reply:
It's the golang formatter